src/main/java/com/company/presentation/controller/BusinessLogic.java

```diff
@@ -3,246 +3,207 @@
+  private CardFileService cardFileService;
+  private BoardlistFileService boardlistFileService;
+
   private Menu menu;
   private File file;
   private Check check;
-  private Card card;
-  private List<Board> allBoards;
-  private int board_pos;
-  private String list_pos;
+
   private String klammer_auf = "[";
   private String klammer_zu = "]";
   private String minus = "-";
 
-  public BusinessLogic() throws IOException, ClassNotFoundException {
+  private String userInputString;
+  private Print print;
+  private Input input;
+
+  public BusinessLogic() throws IOException {
     //  file = new File("SerTest.bin");
     menu = new Menu();
     check = new Check();
-    allBoards = new ArrayList();
+
     boardService = new BoardService();
     boardlistFileService = new BoardlistFileService();
     cardFileService = new CardFileService();
+    print = new Print(boardlistFileService, boardService, cardFileService);
+    input = new Input();
+
+  }
+
+  public Input getInput() {
+    return input;
+  }
+
+  public void setInput(Input input) {
+    this.input = input;
+  }
+
+  public String getUserInputString() {
+    return userInputString;
+  }
+
+  public void setUserInputString(String userInputString) {
+    this.userInputString = userInputString;
   }
 
   public void runApplication() throws IOException, ClassNotFoundException {
     Util util = new Util();
     util.createFile();
+
     if (!check.fileIsEmpty(util.getFile())) {
-      boardService.setAllBoards((List) deserializeObjeckt(util.getFile()));
+      boardService.getScreen().setAllBoards((ArrayList<Board>) deserializeObjeckt(util.getFile()));
     }
 
-    while (!String.valueOf(Menupoints.END.getAction()).equals(input)) {
+    while (!String.valueOf(Menupoints.END.getAction()).equals(userInputString)) {
 
-      input = getInput();
-      validateUserInput(input);
+      userInputString = input.getInput();
+      validateUserInput();
     }
+    boardSerialize(util);
+  }
+
+  public void boardSerialize(Util util) throws IOException {
 
-    if (boardService.getAllBoards() != null) {
-      serializeObjeckt(boardService.getAllBoards(), util.getFile());
+    if (boardService.getScreen().getAllBoards() != null) {
+      serializeObjeckt(boardService.getScreen().getAllBoards(), util.getFile());
       System.out.println("Boardes are saved");
     }
 
     System.out.println("Programm was aboarded");
   }
 
-
-  public void create(Object object) throws IOException {
-    serializeObjeckt(object, file);
-  }
-
-  public void update() throws IOException, ClassNotFoundException {
-
-    Board board = (Board) deserializeObjeckt(file);
-    // board.addBoardlist(boardlist);
-    System.out.println(board.toString());
-  }
-
-  public void delete(Object object) {
-
-  }
-
-  public void validateUserInput(String input) throws IOException, ClassNotFoundException {
+  public void validateUserInput() throws IOException {
     String str_menu = klammer_auf + "1" + minus + "3" + klammer_zu;
```

Comment:
this is a regexp, so please define it like `CORRECT_ANSWER_REGEXP="[1-3]"` and make it static final, then it wil lbe more obvios what is it